DuckDuckGo/AppDelegate.swift

```diff
@@ -543,8 +535,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
 
         if vpnFeatureVisibility.shouldShowThankYouMessaging() && !tunnelDefaults.vpnEarlyAccessOverAlertAlreadyShown {
-            presentVPNEarlyAccessOverAlert()
```

Comment:
I've chosen to keep this PR as simple as possible given that we'll be merging this late in the week. Also, the rest of the logic should stay as we still want waitlist users to have their VPN deactivated upon upgrade, it's just the modal itself that should not appear.